lib/floe/workflow/runner.rb

```diff
@@ -5,29 +5,42 @@ class Workflow
     class Runner
       include Logging
 
-      TYPES         = %w[docker podman kubernetes].freeze
       OUTPUT_MARKER = "__FLOE_OUTPUT__\n"
 
       def initialize(_options = {})
       end
 
+      @runners = {}
       class << self
-        attr_writer :docker_runner
+        # deprecated -- use Floe.set_runner instead
```

Comment:
yard has a deprecated tag we can use ([docs](https://rubydoc.info/gems/yard/file/docs/Tags.md#deprecated))

```suggestion
        # @deprecated Use Floe.set_runner instead
```